com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKeyboard.cs

```diff
@@ -604,51 +624,81 @@ public void Clear()
 
         private void ShowAlphaKeyboardUpperSection()
         {
-            alphaKeysSection?.SetActive(true);
+            if (alphaKeysSection != null)
+            {
+                alphaKeysSection.SetActive(true);
+            }
         }
 
         private void ShowAlphaKeyboardDefaultBottomKeysSection()
         {
-            if (defaultBottomKeysSection?.transform.parent.gameObject.activeSelf == false)
+            if (defaultBottomKeysSection != null)
             {
-                defaultBottomKeysSection?.transform.parent.gameObject.SetActive(true);
+                if (!defaultBottomKeysSection.transform.parent.gameObject.activeSelf)
+                {
+                    defaultBottomKeysSection.transform.parent.gameObject.SetActive(true);
+                }
+                defaultBottomKeysSection.SetActive(true);
             }
-            defaultBottomKeysSection?.SetActive(true);
         }
 
         private void ShowAlphaKeyboardEmailBottomKeysSection()
         {
-            if (emailBottomKeysSection?.transform.parent.gameObject.activeSelf == false)
+            if (emailBottomKeysSection != null)
             {
-                emailBottomKeysSection?.transform.parent.gameObject.SetActive(true);
+                if (!emailBottomKeysSection.transform.parent.gameObject.activeSelf)
+                {
+                    emailBottomKeysSection.transform.parent.gameObject.SetActive(true);
+                }
+                emailBottomKeysSection.SetActive(true);
             }
-            emailBottomKeysSection?.SetActive(true);
         }
 
         private void ShowAlphaKeyboardURLBottomKeysSection()
         {
-            if (urlBottomKeysSection?.transform.parent.gameObject.activeSelf == false)
+            if (urlBottomKeysSection != null)
```

Comment:
The typical code pattern used in MRTK is to avoid multiple if statement.   

Please combine to a single if statement using &&   

``if (urlBottomKeysSection != null && !urlBottomKeysSection.transform.parent.gameObject.activeSelf)
{
}
``

Same comment for other instances